src/binaryutils.cpp

```diff
@@ -73,8 +73,8 @@ static bool isPrecisionEqual(int matPrecision, ovms::Precision tensorPrecision)
 
 static cv::Mat convertStringToMat(const std::string& image) {
     OVMS_PROFILE_FUNCTION();
-    std::vector<unsigned char> data(image.begin(), image.end());
-    cv::Mat dataMat(data, true);
```

Comment:
[Mat](https://docs.opencv.org/4.x/d3/d63/classcv_1_1Mat.html#a23b182c4ffd46abe38f460c1480ae887) (const std::vector< _Tp > &vec, bool copyData=false)

_copyData_:
Flag to specify whether the underlying data of the STL vector should be copied to (true) or shared with (false) the newly constructed matrix. When the data is copied, the allocated buffer is managed using Mat reference counting mechanism. While the data is shared, the reference counter is NULL, and you should not deallocate the data until the matrix is destructed.

Why we do not copy it anymore?